docs/sources/setup/docker.md

```diff
@@ -95,9 +95,6 @@ services:
     image: grafana/beyla:latest
     pid: "service:goblog"
     privileged: true
-    # If using the above capability fails to instrument your service, remove it
-    # and uncomment the line below
-    # privileged: true
```

Comment:
Hi team, this comment seems misplaced since right above there is already `privileged: true` uncommented there. Can you confirm this was intentional?

Reply:
Hi @fridgepoet ! It is certainly a mistake and these 3 lines need to be removed.

Thanks for catching it!